bitnami/mongodb/templates/_helpers.tpl

```diff
@@ -6,6 +6,29 @@ Expand the name of the chart.
 {{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" -}}
 {{- end -}}
 
+{{/*
+Allow the release namespace to be overridden for multi-namespace deployments in combined charts.
+*/}}
+{{- define "mongodb.namespace" -}}
+    {{- if .values.global -}}
```

Comment:
This throws an error in our CI/CD as it should be `.Values` instead of`.values`. Note the `V` vs `v`